File: Store/StoreAPI/Models/Photo.cs
using System;
using System.Collections.Generic;

namespace StoreAPI.Models;

public partial class Photo
{
    public long PhotoId { get; set; }

    public long? ProductId { get; set; }

    public long? CategoryId { get; set; }

    public long? SeriesId { get; set; }

    public long? FirmId { get; set; }

    public string Name { get; set; } = null!;

    public virtual Category? Category { get; set; }

    public virtual Firm? Firm { get; set; }

    public virtual Product? Product { get; set; }

    public virtual Series? Series { get; set; }
}

File: Store/StoreTgBot/src/models/photo.py
from pydantic.v1 import BaseModel
from typing import List, Optional


class RequestCreatePhoto(BaseModel):
    name: str
    product_id: Optional[int] = None
    category_id: Optional[int] = None
    series_id: Optional[int] = None
    firm_id: Optional[int] = None

    def __str__(self):
        return (
            f"name: {self.name}\n"
            f"product_id: {self.product_id}\n"
            f"category_id: {self.category_id}\n"
            f"series_id: {self.series_id}\n"
            f"firm_id: {self.firm_id}\n"
        )
